src/_P013_HCSR04.ino

```diff
@@ -42,12 +44,12 @@ boolean Plugin_013(byte function, struct EventStruct *event, String& string)
       {
         Device[++deviceCount].Number = PLUGIN_ID_013;
         Device[deviceCount].Type = DEVICE_TYPE_DUAL;
-        Device[deviceCount].VType = Sensor_VType::SENSOR_TYPE_SINGLE;
+        Device[deviceCount].VType = SENSOR_TYPE_DUAL;
         Device[deviceCount].Ports = 0;
         Device[deviceCount].PullUpOption = false;
         Device[deviceCount].InverseLogicOption = false;
         Device[deviceCount].FormulaOption = true;
-        Device[deviceCount].ValueCount = 1;
+        Device[deviceCount].ValueCount = 2;
```

Comment:
You should leave the default to 1, as it is used by all instances out there.
And make it configurable to select a different number of outputs.
See the P026_Sysinfo.ino for an example and I also added the new Plugin functions to the Pxxx_Template.ino.

For example, also add this:
```c++
     Device[deviceCount].OutputDataType = Output_Data_type_t::Simple;
```

Reply:
ok, I'll have a look at it